.github/workflows/release-pull-request.yaml

```diff
@@ -11,18 +11,18 @@ jobs:
       matrix:
         image:
           [
-            "quantum-serverless-gateway:latest",
-            "quantum-serverless-ray-node:latest-py39",
-            "quantum-serverless-ray-node:latest-py310",
+            "qiskit-serverless-gateway:latest",
+            "qiskit-serverless-ray-node:latest-py39",
+            "qiskit-serverless-ray-node:latest-py310",
           ]
         include:
-          - image: "quantum-serverless-gateway:latest"
+          - image: "qiskit-serverless-gateway:latest"
```

Comment:
The proxy is missing here :-(.  Would you add it?

Reply:
Ups, totally. Thanks Aki.